docker-compose.yml

```diff
@@ -2,10 +2,9 @@ version: "3"
 
 services:
   mindwendel:
-    container_name: mindwendel
     build:
       context: .
-      target: development
+      target: app
```

Comment:
i dont mind having development here, but maybe add a comment how to switch to app.